R/stanmodels.R

```diff
@@ -17,7 +17,7 @@ Rcpp::loadModule("stan_fit4tune_inv_gamma_mod", what = TRUE)
 # instantiate each stanmodel object
 stanmodels <- sapply(stanmodels, function(model_name) {
   # create C++ code for stan model
-  stan_file <- if (dir.exists("stan")) "stan" else file.path("inst", "stan")
```

Comment:
don't edit stanmodels (its automated)

Reply:
Yes, I do, I just confirmed. I was surprised that file kept getting edited.